config/config.go

```diff
@@ -488,10 +489,6 @@ func (c *Config) Parse() error {
 		return err
 	}
 
-	if c.KubernetesEnableEastWest {
-		log.Warn(`"kubernetes-enable-east-west" parameter is deprecated. Check the "kubernetes-east-west-range" feature`)
```

Comment:
Here the wrong name `kubernetes-enable-east-west` was used, should be `enable-kubernetes-east-west`